<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcountorTuontiDialog</class>
 <widget class="QDialog" name="ProcountorTuontiDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tilinavauksen tuonti Procountorista</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Lisää tuotavat tiedostot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>38</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="valitseTiedostoNappi">
       <property name="text">
        <string>Valitse tiedosto...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../pic/pic.qrc">
         <normaloff>:/pic/kansiossa.png</normaloff>:/pic/kansiossa.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="edellinenGroup">
       <property name="title">
        <string>Edellinen tilikausi</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0">
        <item row="0" column="0">
         <widget class="QLabel" name="edKausiLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="edellinenEiTasmaa">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 85, 0);</string>
          </property>
          <property name="text">
           <string>Debet ja kredit 
eivät täsmää</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>- Tuloslaskelma</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="edTulosCheck">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="edTaseCheck">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>- Tase</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Nykyinen tilikausi</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QLabel" name="nykTaseCheck">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>- Tuloslaskelma</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="nykTulosCheck">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../../pic/pic.qrc">:/pic/ok.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="oikaiseTaseCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Oikaise taseesta 
edellinen tilikausi</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="nykyinenEiTasmaa">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 85, 0);</string>
          </property>
          <property name="text">
           <string>Debet ja kredit 
eivät täsmää</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="nykyKausiLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>- Tase</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
